Packs/TAXIIServer/ReleaseNotes/2_1_12.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### TAXII2 Server
+
+- Added the extensions field in STIX Domain Object (SDO) indicators. The server now includes custom XSOAR indicator fields as extension properties when serving indicators.
```

Comment:
Added the **Cortex XSOAR Extensions fields** in STIX Domain Object (SDO) indicators. Cortex XSOAR now includes custom XSOAR indicator fields as extension properties when generating indicators.